test/verify/check-storage-stratis

```diff
@@ -0,0 +1,370 @@
+
+        b.wait_in_text('#detail-sidebar', dev_3)
+        b.wait_in_text('#detail-sidebar .sidepanel-row:contains(%s)' % dev_3, "data")
+
+        # Add a cache blockdev
+        b.click('#detail-sidebar .pf-c-card__actions button')
+        self.dialog({'tier': "cache",
+                     'disks': {dev_4: True}})
+        b.wait_in_text('#detail-sidebar', dev_4)
+        b.wait_in_text('#detail-sidebar .sidepanel-row:contains(%s)' % dev_4, "cache")
+
+        # Rename the pool
+        b.click('#detail-header button:contains(Rename)')
+        self.dialog({'name': "pool0-renamed"})
+        b.wait_in_text('#detail-header', "pool0-renamed")
+        b.wait_in_text('#detail-content', "/dev/stratis/pool0-renamed/fsys2")
+
+        # Destroy the pool
+        b.click('#detail-header button:contains(Delete)')
+        self.dialog_wait_open()
+        b.assert_pixels('#dialog', "delete-pool")
+        self.dialog_apply()
+        self.dialog_wait_close()
+        b.wait_visible("#storage")
+        b.wait_not_in_text("#devices", "pool0-renamed")
+
+        # Check that the entries have disappeared from fstab
+        self.assertEqual(m.execute("grep /run/fsys1 /etc/fstab || true"), "")
+        self.assertEqual(m.execute("grep /run/fsys2 /etc/fstab || true"), "")
+
+    def testEncrypted(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/storage")
+
+        dev_1 = "/dev/sda"
+        m.add_disk("4G", serial="DISK1")
+        b.wait_in_text("#drives", dev_1)
+
+        dev_2 = "/dev/sdb"
+        m.add_disk("4G", serial="DISK2")
+        b.wait_in_text("#drives", dev_2)
+
+        # Create an encrypted pool with a filesystem, but don't mount it
+        self.dialog_open_with_retry(trigger=lambda: self.devices_dropdown("Create Stratis pool"),
+                                    expect=lambda: (self.dialog_is_present('disks', dev_1) and
+                                                    self.dialog_check({"name": "pool0"})))
+        self.dialog_set_val("encrypt.on", True)
+        self.dialog_set_val("passphrase", "foodeeboodeebar")
+        self.dialog_set_val("passphrase2", "foodeeboodeebar")
+        self.dialog_set_val("disks", {dev_1: True})
+        b.assert_pixels("#dialog", "create-encrypted-pool")
+        self.dialog_apply()
+        self.dialog_wait_close()
+
+        b.wait_in_text("#devices", "pool0")
+        b.click('.sidepanel-row:contains("pool0")')
+        b.wait_visible('#storage-detail')
+        b.wait_in_text('#detail-header dt:contains(Key description) + dd', "pool0")
+        b.click("button:contains(Create new Filesystem)")
+        self.dialog({'name': 'fsys1',
+                     'mount_options.auto': False,
+                     'mount_point': '/run/fsys1'})
+        b.wait_in_text("#detail-content", "fsys1")
+
+        # Check that it has an entry in fstab and that it is "noauto"
+        self.assertIn("noauto", m.execute("grep /run/fsys1 /etc/fstab"))
```

Comment:
Does this also appear in crypttab? or is this handled internally?

Reply:
No, this does not appear in crypttab.  Stratis handles all details about encrypted block devices of its pools.